tools/appveyor.psm1

```diff
@@ -108,6 +108,20 @@ function Invoke-AppVeyorInstall
         Update-AppveyorBuild -message $buildName
     }
 
+    # Generate credentials for appveyor remoting tests.
```

Comment:
@TravisEz13 I have implemented the change and tested offline using Invoke-AppVeyorFull.  However, the test still fails with Access Denied.  This is probably because the LocalAccountTokenFiliterPolicy is not set.  I'll update the code to set it as needed.